package.json

```diff
@@ -32,8 +32,9 @@
   "devDependencies": {
     "@biomejs/biome": "^1.9.4",
     "@zowe/secrets-for-zowe-sdk": "^8.10.4",
+    "tsx": "^4.19.2",
+    "ssh-config": "^5.0.3",
     "husky": "^9.1.7",
     "rimraf": "^6.0.1",
```

Comment:
Please remove the trailing comma - `npm install` emits an error about invalid syntax